pkg/artifactory/resource/repository/local/resource_artifactory_local_repository_test.go

```diff
@@ -520,6 +520,68 @@ func TestAccLocalNugetRepository(t *testing.T) {
 	})
 }
 
+func TestAccLocalTerraformModuleRepository(t *testing.T) {
+
+	_, fqrn, name := acctest.MkNames("terraform-local", "artifactory_local_terraform_module_repository")
+	params := map[string]interface{}{
+		"registry_type": "module",
+		"name":          name,
+	}
+	localRepositoryBasic := acctest.ExecuteTemplate("TestAccLocalTerraformModuleRepository", `
```

Comment:
Going forward, I think it would be more readable to have this be proper multi-line. Parts of this function call are multi-line and parts aren't

Reply:
Not sure what you don't like there. Can you point me to the example of how you think it should be?

Reply:
```go
thing := acctest.ExecuteTemplate(
    "name",
    "my_hcl",
    params
)